src/plasma/plasma_manager.cc

```diff
@@ -765,35 +779,27 @@ void process_transfer_request(event_loop *loop,
     return;
   }
 
+  /* Allocate and append the request to the transfer queue. */
+  ObjectBuffer obj_buffer;
```

Comment:
Now that we have the new PascalCase naming convention, let's try to move away from these abbreviations and write object_buffer instead!